charts/jenkins/templates/jenkins-controller-ingress.yaml

```diff
@@ -1,12 +1,6 @@
 {{- $kubeTargetVersion := default .Capabilities.KubeVersion.GitVersion .Values.kubeTargetVersionOverride }}
 {{- if .Values.controller.ingress.enabled }}
-{{- if semverCompare ">=1.19-0" $kubeTargetVersion -}}
-apiVersion: networking.k8s.io/v1
```

Comment:
this isn't what i meant, i meant just remove it from values.yaml, there's no reason to have it configurable